infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/SessionManager.java

```diff
@@ -323,6 +324,8 @@ private void createOutgoingSessionsBySubscriber(ReplicationSubscriber subscriber
         log.info("Total of {} outgoing sessions created with subscriber {}, sessions={}", sessionsToAdd.size(),
                 subscriber, sessionsToAdd);
 
+        configManager.getSessionLockMap().putAll(sessionsToAdd.stream()
```

Comment:
Please add a comment about this code section. Why it is needed? I see it being used inside createOutgoingSessionsBySubscriber, createIncomingSessionsBySubscriber and not other create sessions method?